tools/java/org/unicode/cldr/util/StandardCodes.java

```diff
@@ -1073,6 +1075,17 @@ boolean isWellFormed(String candidate) {
         public String toCompatString() {
             return this == region ? "territory" : toString();
```

Comment:
Another way to do this is to store the compat string as a field, but either way it would be nice to see a comment on this public method explaining its *not* the enum name.

Reply:
done (the doc, and some other changes)